.tests.bash

```diff
@@ -21,6 +21,11 @@ for example in ${EXAMPLES} ; do
 
 	"${MAKE}" packages
 
+	# Ensure we produce runnable binaries on current platform
+	for b in build/bin/* ; do
+		"$b"
```

Comment:
if the binary actually does something without parameters this might have unintended consequences, for example a server that runs forever